.gitignore

```diff
@@ -12,6 +12,12 @@ book
 # Used for deployments
 firebase-tmp/
 
+# Nix build output goes here
+result
+
+# Direnv files
+.direnv
+
```

Comment:
Matching comment above, add `.envrc` to the gitignore—but _not_ `.envrc.example`, which will be version-controlled.